packages/extension-chakra-storefront/src/middleware/passwordless.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {Router, Request, Response, NextFunction} from 'express'
+import {validateSlasCallbackToken} from '../utils/slas'
+import {emailLink} from '../utils/email'
+
+interface PasswordlessOptions {
+    callbackURI?: string
+    landingPath?: string
+    shortCode: string
+    tenantId: string
+}
+
+const asyncHandler =
+    (fn: (req: Request, res: Response, next: NextFunction) => Promise<void>) =>
+    (req: Request, res: Response, next: NextFunction) => {
+        Promise.resolve(fn(req, res, next)).catch(next)
+    }
+
+const passwordlessMiddleware = (options: PasswordlessOptions) => {
+    const {
+        callbackURI = '/passwordless-login-callback',
+        landingPath = '/passwordless-login-landing',
+        shortCode = '',
+        tenantId = ''
```

Comment:
These options are required, so we don't need default values for them. 

The other middleware (for password reset) already does it correctly.

```suggestion
        shortCode,
        tenantId
```